rollup.config.js

```diff
@@ -60,13 +60,15 @@ const buildUmd = ({ env }) => ({
     sourceMaps(),
     sizeSnapshot(),
     env === 'production' &&
-      uglify({
+      terser({
+        sourcemap: true,
         output: { comments: false },
         compress: {
           keep_infinity: true,
           pure_getters: true,
         },
         warnings: true,
+        ecma: 5,
         toplevel: false,
```

Comment:
@TrySound should we run terser on CJS production build as well? I see that it can accept a `format` option.

Reply:
Yeah, probably